tests/system/large/functions/test_remote_function.py

```diff
@@ -1359,6 +1360,46 @@ def square_num(x):
         )
 
 
+@pytest.mark.flaky(retries=2, delay=120)
+def test_remote_function_warns_default_cloud_function_service_account(scalars_dfs):
+    project = "bigframes-dev-perf"
+
+    rf_session = bigframes.Session(context=bigframes.BigQueryOptions(project=project))
+
+    try:
+        with pytest.warns(FutureWarning, match="You have not explicitly set a"):
+            warnings.simplefilter("always")
+
+            @rf_session.remote_function(
```

Comment:
I think we don't actually have to deploy a function to test this warning. Will try something and add a commit if so.